packages/ui5-application-writer/templates/optional/typescript/tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "rootDir": "webapp",
     "outDir": "./dist",
     "baseUrl": "./",
-    "paths": {
+    "paths": {<% if (app.id.includes('.')) { %>
+      "<%=app.id.replace(/\./g, '/')%>/*": [
+        "webapp/*"
+      ],
+      <% } %>
       "<%- app.id %>/*": [
```

Comment:
Why did you leave the old one? Shouldn't this one be replaced with your new code?

Reply:
The idea behind introducing a second path with the IF statement, was to ensure that other references who require `myname.test.namespace` continue to work. I didnt want to do a full replace on the chance this introduced a regression. 
The `component.ts` file in Freestyle writer is the only TS that does the replace;
```
import { createDeviceModel } from "<%=app.id.replace(/\./g, '/')%>/model/models";
```
However, there are other TS files that just reference the app.id;
```
name: "<%- app.id %>.view.ViewSettingsDialog",
```

Reply:
But that is inconsistent, and we shouldn't do that. To be honest, within the same project the references should all be relative e.g. `import { createDeviceModel } from "./model/models";`
Please clean this up properly, `"<%=app.id.replace(/\./g, '/')%>/*": [ "webapp/*" ]` is the correct line for all projects (it was simply wrong before) an also make sure that in our templates we always use relative paths.

Btw, where is `name: "<%- app.id %>.view.ViewSettingsDialog"`?

Reply:
@petermuessig please check my statements above and let us know if you disagree.

Reply:
@tobiasqueck the reference file https://github.com/SAP/open-ux-tools/blob/main/packages/fiori-freestyle-writer/templates/listdetail/add/webapp/controller/List.controller.ts#L147

Reply:
That is a property that we are giving to the fragment loader. That is not related to typescript. UI5 needs the ID of the fragment and does its own resolution of the file based on the registered path for the corresponding namespace/module.

Reply:
@jlongie I checked the code,  `import { createDeviceModel } from "<%=app.id.replace(/\./g, '/')%>/model/models";` seems the only one that needs fixing and should be `import { createDeviceModel } from "./model/models";`.
The rest is good to go.